client/app/cloud/project/compute/infrastructure/virtualMachine/add/cloud-project-compute-infrastructure-virtualMachine-add.html

```diff
@@ -332,7 +338,8 @@
             <div class="cui-grid-list"
                  data-ng-if="$ctrl.submitted.step3 && $ctrl.model.flavor">
                 <div class="cui-grid-cell">
-                    <oui-radio data-model="$ctrl.model.flavor"
+                    <oui-radio class="oui-radio_thumbnail-light"
```

Comment:
Maybe we could change `oui-radio` to have a `data-variant="light"`, just like we do with `oui-select-picker` :)

Reply:
This is done :)